source/NeutronaWand/NeutronaWand.ino

```diff
@@ -5799,6 +5917,112 @@ void wandSerialSend(int i_message) {
   wandComs.sendDatum(sendStruct);
 }
 
+#ifdef GPSTAR_NEUTRONA_WAND_PCB
+  void clearEEPROM() {
+    // Clear out the EEPROM only in the memory addresses used for our EEPROM data object.
+    for(unsigned int i = 0 ; i < sizeof(objEEPROM); i++) {
+      EEPROM.put(i, 0);
+    }
+
+    updateCRCEEPROM();
+  }
+
+  void saveEEPROM() {
+    // (Video Game Modes) + Cross The Streams / Cross The Streams Mix / Over Heating    
+    uint8_t i_cross_the_streams = 1;
```

Comment:
Would there be any benefit in just having a singular variable for the "firing mode" which is a byte to represent the current 3 modes and any future modes? eg. VG mode = 1, CTS = 2, CTS_Mix = 3, etc. rather than tracking 2 or more separate values? The available options could then be handled as ENUM values which I believe are bytes or at least unsigned int's? That way if you happen to add more options to this space later you're still only reading/writing a single value.

Reply:
Hmm perhaps, but cross the streams and mix versions are user configurable editable values, so it is easier for people to set it to true or false if they are manually flashing the boards vs having people mess around with enums and or using more memory by using both enums and the booleans.

Reply:
With your inclusion of the EEPROM menus, you've effectively made the user-configurable values a feature exclusive for advanced users--those who wouldn't understand an ENUM no longer need to make those changes. It's probably more of a line in the sand to be drawn for what type of users you expect to be digging into the software vs. ease of maintenance.

Reply:
The more I've thought about this, It's a non-issue and has a larger footprint to change the flags involved. Consider this resolved.